dart/.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: dart
```

Comment:
It turns out that the travis testing doesn't support multiple .travel.yml files. :-(

Just remove this file, I'll integrate it into the travis testing.